job/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowJob.java

```diff
@@ -100,6 +101,8 @@
     private transient LazyBuildMixIn<WorkflowJob,WorkflowRun> buildMixIn;
     /** defaults to true */
     private @CheckForNull Boolean concurrentBuild;
+    
+    public static final AlternativeUiTextProvider.Message<WorkflowJob> BUILD_NOW_TEXT = new AlternativeUiTextProvider.Message<WorkflowJob>();
```

Comment:
Because it is generalizing a field already named `BUILD_NOW_TEXT`, which is the text normally called (in English) _Build Now_.